object/search_test.go

```diff
@@ -218,6 +218,29 @@ func TestSearchFilters_AddSplitIDFilter(t *testing.T) {
 	})
 }
 
+func TestSearchFilters_AddFirstIDFilter(t *testing.T) {
+	id := testOID()
+
+	fs := new(object.SearchFilters)
+	fs.AddFirstObjectIDFilter(object.MatchStringEqual, id)
```

Comment:
i doubt (quite a bit) that from the outside such an interface will be understandable without the "split" word. Now it reads like "find 1st object" - too abstract imo

my suggestion is
```suggestion
	fs.AddFirstSplitObjectFilter(object.MatchStringEqual, id)
```
or any other alternative with `Split` substr